vector/src/main/res/layout/item_verification_big_image.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<im.vector.app.core.ui.views.ShieldImageView xmlns:android="http://schemas.android.com/apk/res/android"
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:contentDescription="@string/trusted"
```

Comment:
`android:contentDescription` -> `tools:contentDescription`

Also please format the file.

Reply:
done